src/Magento/FunctionalTestingFramework/Suite/Util/SuiteObjectExtractor.php

```diff
@@ -71,28 +81,80 @@ public function parseSuiteDataIntoObjects($parsedSuiteData)
 
             $this->validateSuiteName($parsedSuite);
 
-            //extract include and exclude references
-            $groupTestsToInclude = $parsedSuite[self::INCLUDE_TAG_NAME] ?? [];
-            $groupTestsToExclude = $parsedSuite[self::EXCLUDE_TAG_NAME] ?? [];
+            try {
+                // extract include and exclude references
+                $groupTestsToInclude = $parsedSuite[self::INCLUDE_TAG_NAME] ?? [];
+                $groupTestsToExclude = $parsedSuite[self::EXCLUDE_TAG_NAME] ?? [];
 
-            //resolve references as test objects
-            $includeTests = $this->extractTestObjectsFromSuiteRef($groupTestsToInclude);
-            $excludeTests = $this->extractTestObjectsFromSuiteRef($groupTestsToExclude);
+                // resolve references as test objects
+                // continue if failed in include
+                $include = $this->extractTestObjectsFromSuiteRef($groupTestsToInclude);
+                $includeTests = $include['objects'] ?? [];
+                $stepError = $include['status'] ?? 0;
+                $includeMessage = '';
+                if ($stepError != 0) {
+                    $includeMessage = "ERROR: " . strval($stepError) . " test(s) not included for suite "
+                        . $parsedSuite[self::NAME];
+                }
 
-            // parse any object hooks
-            $suiteHooks = $this->parseObjectHooks($parsedSuite);
+                // it's ok if failed in exclude
+                $exclude = $this->extractTestObjectsFromSuiteRef($groupTestsToExclude);
+                $excludeTests = $exclude['objects'] ?? [];
+
+                // parse any object hooks
+                $suiteHooks = $this->parseObjectHooks($parsedSuite);
+
+                // log error if suite is empty
+                if ($this->isSuiteEmpty($suiteHooks, $includeTests, $excludeTests)) {
+                    LoggingUtil::getInstance()->getLogger(self::class)->error(
```

Comment:
Should unable to parse suite be moved to mftf.log as well, for consistent error logging?
 
Current:
```
NOTICE: 202 Section name violations detected. See mftf.log for details.[]
NOTICE: 617 section element name violations detected. See mftf.log for details.[]
NOTICE: 301 data entity name violations detected. See mftf.log for details.[]
NOTICE: 5 data entity key violations detected. See mftf.log for details.[]
NOTICE: 10 action group name violations detected. See mftf.log for details.[]
ERROR: Unable to parse suite WYSIWYGDisabledSuite
NOTICE: 38 Page name violations detected. See mftf.log for details.[]

ERROR: 1 Suite(s) failed to generate. See mftf.log for details.
ERROR: 7 Test(s) failed to generate. See mftf.log for details.
Generate Tests Command Run (with errors)
```

Expected:
`ERROR: Unable to parse suite WYSIWYGDisabledSuite` only appears in mftf.log

Reply:
We are logging some errors to console before and now, like deprecation message, story/title uniqueness, some extends error, etc. I feel display these errors are useful as long as they are not all over the place. The log, on the other hand, is quite messy and hard to find.

I guess the question is:
Not only this suite message, there are other error messages. Do we want to remove all the error display?

Reply:
I get your point. I'm okay with this too. Looks like there are other error messages as well that get displayed. There many be some inconsistency on which errors are displayed on console and which are not. It may be good to list them down to see which ones are useful to be printed in console and which are not.

Reply:
I think it might be good to remove all error displays. This way we can force users to look at the log for errors.

Reply:
> I think it might be good to remove all error displays. This way we can force users to look at the log for errors.

Implement it as default. Error will print in verbose mode.